### PR TITLE
MSPSDS-596: add links to from user account page

### PR DESCRIPTION
**Pull request by raamSoftwire**

### CODE REVIEW

File: web/app/clients/keycloak_client.rb [hunk 1]
```diff
@@ -32,6 +32,10 @@ def login_url(redirect_uri)
     @client.url_login_redirect(redirect_uri)
   end
 
+  def url_user_account
```
[the remark this answers is not in the document]
Comment (raamSoftwire) [reply web/app/clients/keycloak_client.rb]: Good point, I've changed that now.

File: web/app/views/layouts/application.html.slim [hunk 2]
```diff
@@ -61,9 +61,12 @@ html.govuk-template.app-html-class[lang="en"]
...
           nav
             ul#navigation.govuk-header__navigation[aria-label="Top Level Navigation"]
-              - if user_signed_in? && policy(User).index?
+              - if user_signed_in?
                 li.govuk-header__navigation-item
-                  = link_to "View users", "/users", class: "govuk-header__link"
+                  = link_to "Manage your account", url_user_account, class: "govuk-header__link", target: "_blank"
+                - if policy(User).index?
+                  li.govuk-header__navigation-item
+                    = link_to "View users", "/users", class: "govuk-header__link"
```
[the remark this answers is not in the document]
Comment (raamSoftwire) [reply web/app/views/layouts/application.html.slim]: I've removed the controllers and the views. There is still a user helper, model and a policy, are these OK to leave?

File: docker-compose.yml [hunk 3]
```diff
@@ -27,6 +27,8 @@ services:
     build:
       context: ./keycloak
       dockerfile: Dockerfile
+    volumes:
+    - ./keycloak/govuk-theme/govuk/:/opt/jboss/keycloak/themes/govuk/
```
[the remark this answers is not in the document]
Comment (raamSoftwire) [reply docker-compose.yml]: Maybe, although I'm not completely sure. The `npm run build` command is in the Dockerfile, so running `docker-compose down && docker-compose build && docker-compose up -d` would solve this.

[the remark this answers is not in the document]
Comment (tom-bell-softwire) [reply docker-compose.yml]: I think the build step in the Dockerfile actually copies and builds the theme on the docker app itself, so the build artefacts aren't available to the host machine.

I'd suggest we remove this, or perhaps move it to `docker-compose.debug.yml`, so it isn't part of the default Docker setup.

[the remark this answers is not in the document]
Comment (raamSoftwire) [reply docker-compose.yml]: The build artefacts are copied over so this is fine to leave.

**Pull request merged**